hazelcast-sql/src/main/java/com/hazelcast/sql/impl/QueryUtils.java

```diff
@@ -181,4 +184,13 @@ public static <K, V> MapContainer getMapContainer(IMap<K, V> map) {
         MapServiceContext mapServiceContext = mapService.getMapServiceContext();
         return mapServiceContext.getMapContainer(map.getName());
     }
+
+    public static long parseLong(SqlValidator validator, SqlOption option) {
```

Comment:
query utils seems to contain methods mostly for execution not for parsing. this method would fit better in `com.hazelcast.jet.sql.impl.parse` package

Reply:
Not exactly. `prepareSearchPaths` is used even before query planning; `workerName` is used even on member start. I think this class a okay-ish place for util method.

Reply:
IMO it spaghettifies dependencies between packages, top level `com.hazelcast.sql.impl` will be dependent on deep implementation package `com.hazelcast.jet.sql.impl.parse`, but our code would blow up any dependency analysis tools

Reply:
Good point for servicing planned tech debt

Reply:
Will do it as part of tech debt.

Reply:
You can choose to address technical debt later or never, depending on the complexity and potential benefit. However, you cannot create technical debt if the proper solution is as easy as the proposed solution.